internal/db/src/schema/keys.ts

```diff
@@ -152,3 +156,11 @@ export const encryptedKeysRelations = relations(encryptedKeys, ({ one }) => ({
     references: [workspaces.id],
   }),
 }));
+
+export const keyMigrations = mysqlTable("key_migrations", {
+  id: varchar("id", { length: 256 }).primaryKey(),
+  workspaceId: varchar("workspace_id", { length: 256 }).notNull(),
+  algorithm: mysqlEnum("algorithm", [
```

Comment:
Let's not use enums for this, but just a raw string; having to edit the DB each time to support a new algorithm sounds unnecessary.